src/components/shop.ts

```diff
@@ -16,12 +16,23 @@ export class NimiqShop extends LitElement {
 
   connectedCallback() {
     super.connectedCallback()
-    // set address to checkout buttons
-    let buttons = this.getElementsByTagName('nimiq-shop-checkout-button')
+    const buttons = this.getElementsByTagName('nimiq-shop-checkout-button')
     for (let i = 0; i < buttons.length; i++) {
-      buttons[i].setAttribute('shop-address', this.address)
+      buttons[i].addEventListener('checkout', this.checkout);
```

Comment:
```suggestion:-3
  for (const button of this.getElementsByTagName('nimiq-shop-checkout-button')) {
    button.addEventListener('checkout', this.checkout);
```
Again just, an idea. Please let me know what you think about suggestions like this.

Reply:
GitHub does not yet support multi-line suggestions. GitLab does. :-S
Move to GitLab? :D